okio/jvm/src/main/java/okio/ByteString.kt

```diff
@@ -376,7 +376,7 @@ internal constructor(
   @Throws(IOException::class)
   private fun readObject(`in`: ObjectInputStream) {
     val dataLength = `in`.readInt()
-    val byteString = ByteString.read(`in`, dataLength)
+    val byteString = `in`.read(dataLength)
```

Comment:
Hmm we should probably name this `readByteString` in Kotlin and keep `ByteString.read` only for Java.